### PR TITLE
cosmos-sdk ut coverage improve

### PR DESCRIPTION
**Pull request by Ywmet**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okexchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: libs/cosmos-sdk/baseapp/baseapp_paralled_test.go [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+	}
+
+	chainA, chainB := NewChain(env), NewChain(env)
+
+	testCases := []struct {
+		name     string
+		malleate func(t *testing.T, chain *Chain, isParallel bool) ([]byte, []byte)
+	}{
+		{
+			"five txs one group:a->b b->c c->d d->e e->f",
```
Comment (lcmmhcc) [inline libs/cosmos-sdk/baseapp/baseapp_paralled_test.go:231]: maybe add some scenarios like tx failed

File: libs/cosmos-sdk/x/capability/types/types_test.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+			owner = types.NewOwner("bank", fmt.Sprintf("send-%d", i))
+		} else {
+			owner = types.NewOwner("slashing", fmt.Sprintf("slash-%d", i))
+		}
+
+		owners[i] = owner
+		require.NoError(t, co.Set(owner))
+	}
+
+	sort.Slice(owners, func(i, j int) bool { return owners[i].Key() < owners[j].Key() })
```
Comment (lcmmhcc) [inline libs/cosmos-sdk/x/capability/types/types_test.go:43]: test  owner already claimed

File: libs/cosmos-sdk/x/capability/keeper/keeper_test.go [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+	got, ok := sk2.GetCapability(suite.ctx, "transfer")
+	suite.Require().False(ok)
+	suite.Require().Nil(got)
+
+	suite.Require().NoError(sk1.ReleaseCapability(suite.ctx, cap1))
+	got, ok = sk1.GetCapability(suite.ctx, "transfer")
+	suite.Require().False(ok)
+	suite.Require().Nil(got)
+
+	suite.Require().Error(sk1.ReleaseCapability(suite.ctx, nil))
```
Comment (lcmmhcc) [inline libs/cosmos-sdk/x/capability/keeper/keeper_test.go:261]: need to check prefixStore's indexkey when update

File: libs/cosmos-sdk/x/capability/types/types_test.go (hunk 2 above)
Comment (Ywmet) [reply libs/cosmos-sdk/x/capability/types/types_test.go:43]: already claimed

File: libs/cosmos-sdk/x/capability/keeper/keeper_test.go (hunk 3 above)
Comment (Ywmet) [reply libs/cosmos-sdk/x/capability/keeper/keeper_test.go:261]: image
When firstly call ReleaseCapability ,2nd GetCapablily mean the same with check prefixStore's indexkey when update.

File: libs/cosmos-sdk/baseapp/baseapp_paralled_test.go (hunk 1 above)
Comment (Ywmet) [reply libs/cosmos-sdk/baseapp/baseapp_paralled_test.go:231]: 0fd6a96a9a27899dd2a143afb26f140b6b7d022e has add two case with conflicted in contract

**Review by lcmmhcc — APPROVED** [review]

**Review by scf0220 — APPROVED** [review]

**Pull request merged**